GoogleForCreators/web-stories-wp
# Hierarchal checkbox checked attribute

## Context



[See pr 9150](https://github.com/google/web-stories-wp/pull/9150#discussion_r716187510) 

When the checkbox input is checked/unchecked the attribute is not updated on the DOM's input element. 
## Summary



## Relevant Technical Choices
Force a rerender by using checked status as part of the component's `key` value. 


## To-do


n/a

## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Check / uncheck a checkbox within the taxonomy hierarchal category selections. 
2. Look at the html element in dev tools to see if the `checked` attribute is reflected correctly. 

https://user-images.githubusercontent.com/1820266/135165316-fd731b7b-d771-4f1a-bd39-3fdbbf372b00.mov



## Reviews

### Does this PR have a security-related impact?
no



### Does this PR change what data or activity we track or use?
no


### Does this PR have a legal-related impact?
no


## Checklist




---



Partially Fixes #9149
